### ISSUE TITLE
Implement baseline WCAG fixes that are suggested by Firefox's accessibility inspector

### ISSUE THREAD

**Issue by BryceStevenWilley** [SuffolkLITLab/docassemble-AssemblyLine#146]
Just came across this today, though it's definitely been in for a while: Firefox's developer tools have an [Accessibility inspector tab](https://developer.mozilla.org/en-US/docs/Tools/Accessibility_inspector) that shows Contrast, Keyboard, and Text Label issues on the current page, with violations having links to https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG. Screenshot of example issues on the Weaver attached.

![Screenshot from 2021-05-11 19-26-11](https://user-images.githubusercontent.com/6252212/117896687-178b3a80-b28f-11eb-807e-37e6d874d196.png)

Most of the Text Label and Keyboard issues will have to be fixed in docassemble core, but we could probably fix the contrast in here (or Mass Access?).

**Comment by plocket** [comment]
See https://github.com/jhpyle/docassemble/issues/113 for a request for an accessible bootstrap theme. I'm doing a little looking around for one.

**Comment by nonprofittechy** [comment]
Fixed upstream
